website_seo_redirection/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# © 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# © 2018 Juan Carlos Montoya <jcarlosmontoyach@gmail.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+{
+    "name": "Website SEO Redirection",
+    "summary": "Redirect any controller to the URL of your dreams",
+    "version": "11.0.1.0.0",
+    "category": "Website",
+    "website": "https://tecnativa.com/",
```

Comment:
@jcarlosmontoya Change URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)